react/new/src/components/MainNavigation/index.tsx

```diff
@@ -1,9 +1,15 @@
 import { Menu, Group, Center, Burger, Container } from '@mantine/core'
 import { useDisclosure } from '@mantine/hooks'
-import { IconChevronDown } from '@tabler/icons-react'
+import {
```

Comment:
Curious: Are multi-line imports like this the norm in ts? Coming from the non-web world I kinda hate it but if that's the standard it's fine by me.

Reply:
Why'd 

`import {
  IconChevronDown,
  IconChevronRight,
  IconSettings,
} from '@tabler/icons-react'`

end up multiline then? At least in github (could be improperly rendering, I've seen that before)

The single line `import { Menu, Group, Center, Burger, Container } from '@mantine/core';` is more what I expected.